swift/apple/FirezoneNetworkExtension/DeviceMetadata.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  DeviceMetadata.swift
+//  Firezone
+//
+//  Created by Jamil Bou Kheir on 2/23/24.
+//
```

Comment:
Does XCode do this? Cause Git already tracks all that info, plus it doesn't use mm/dd/yy

Reply:
Yeah, Xcode does that. We probably need a consistent header.